### PR TITLE
Fix issues with single-sign-on button. Add prod deployment config.

### PR DESCRIPTION
**Pull request by rowanhogan**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -10,14 +10,14 @@ steps:
 
   - name: ":shipit: Ship It - Staging"
     command: docker-compose run --rm build bin/deploy $ENVIRONMENT
-    branches: master staging/*
+    branches: master staging/* feature/*
```
Comment (kizza) [inline .buildkite/pipeline.yml]: I'm more used to seeing `release` rather than `feature` (is this a "ps" thingo though :)

File: bin/test [hunk 2]
```diff
@@ -3,4 +3,4 @@
 set -eu pipefail
 
 yarn
-yarn test
+yarn lint
```
Comment (kizza) [inline bin/test]: ...does this mean that it's not gonna run `test`? and only `lint`?

Comment (rowanhogan) [reply bin/test]: There IS only `lint` 🚫 🥄

Comment (kizza) [reply bin/test]: (perhaps `yarn` runs tests by default?? I have no idea :)

Comment (kizza) [reply bin/test]: lol. of course :)

**Pull request merged**